.github/workflows/tests.yml

```diff
@@ -34,13 +34,14 @@ jobs:
       - name: Meilisearch (latest version) setup with Docker
         run: docker run -d -p 7700:7700 getmeili/meilisearch:latest meilisearch --no-analytics --master-key=masterKey
       - name: Test with pytest
-        run: pipenv run pytest --cov-report=xml
-      - name: Upload coverage
-        uses: codecov/codecov-action@v3
-        with:
-          fail_ci_if_error: true
-        env:
-          CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
+        run:
+          pipenv run pytest --cov-report=xml
+          # - name: Upload coverage
+          #   uses: codecov/codecov-action@v3
+          #   with:
+          #     fail_ci_if_error: true
+          #   env:
+          #     CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
```suggestion
```

Reply:
I would go with completely remove them, and mention in the PR in the issue we will open to say we want something else than code coverage